conf/eslint.json

```diff
@@ -116,6 +116,7 @@
         "no-useless-call": "off",
         "no-useless-concat": "off",
         "no-useless-constructor": "off",
+        "no-useless-escape": "off",
```

Comment:
I don't see why we wouldn't want this rule on, but that's probably up to @nzakas to decide.